Doppler.BillingUser/Utils/BillingHelper.cs

```diff
@@ -18,5 +28,101 @@ public static bool IsUpgradePending(UserBillingInformation user, Promotion promo
 
             return user.PaymentMethod != PaymentMethodEnum.CC;
         }
+        public static SapBillingDto MapBillingToSapAsync(IOptions<SapSettings> _sapSettings, string cardNumber, string cardHolderName, BillingCredit billingCredit, UserTypePlanInformation currentUserPlan, UserTypePlanInformation newUserPlan, string authorizationNumber, int invoidId)
```

Comment:
You only need the settings for the TimeZoneOffset, what do you think about simply pass the TimeZoneOffset in order to reduce the coupling?

Also, it is not async.

```suggestion
        public static SapBillingDto MapBillingToSap(DateTimeOffset timeZoneOffset, string cardNumber, string cardHolderName, BillingCredit billingCredit, UserTypePlanInformation currentUserPlan, UserTypePlanInformation newUserPlan, string authorizationNumber, int invoidId)
```

Reply:
Also, there is a typo in `invoidId`